src/modules/registrypreview/RegistryPreviewUI/Strings/en-US/Resources.resw

```diff
@@ -251,4 +251,13 @@
   <data name="ZeroLength" xml:space="preserve">
     <value>(zero-length binary value)</value>
   </data>
+  <data name="InvalidBinary" xml:space="preserve">
+    <value>(Invalid binary value)</value>
+  </data>
+  <data name="InvalidDword" xml:space="preserve">
+    <value>(Invalid DWORD (32-bit) value)</value>
+  </data>
+  <data name="InvalidString" xml:space="preserve">
+    <value>(Invalid string value)</value>
+  </data>
```

Comment:
Please sort these as they go into the resw file: the IDE may do it automatically for you, if opened in the built-in editor.

Reply:
I used the built-in editor to modify the file but the IDE didn't sort them for some reason. Should be fixed.